token/faucet/app.js

```diff
@@ -19,22 +19,26 @@ function runApp(config) {
 
   // Configure rate limiting. Allow at most 1 request per IP every 5 sec.
   const opts = {
-    points: 90,   // Point budget.
+    points: 1,   // Point budget.
     duration: 5, // Reset points consumption every 5 sec.
```

Comment:
If I'm reading this correctly, a single IP can get ~1200 OGN per minute. Because the max boost is 100 OGN, that seems a little excessive. How about 100 or 200 OGN per minute? (if this is scripted, someone would still be able to get 8,640,000 OGN per day, which is a lot)

Reply:
Yes you are correct. That's a bit high. Adjusted to allow 100 OGN per 60 sec.